packages/react/src/toast/toast.stories.tsx

```diff
@@ -72,13 +72,27 @@ const ExampleComponent = () => {
         onClick={() => {
           toast.create({
             title: 'Data submitted!',
+            description: 'Your user data has been submitted successfully!',
             type: 'success',
             placement: 'bottom-start',
           })
         }}
       >
         Add bottom-start toast
       </button>
+      <button
+        onClick={() => {
+          toast.create({
+            render: () => (
```

Comment:
Does the custom `render` receive `props` as in Zag?

Reply:
Yes it does